py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -179,7 +180,14 @@ def aten_ops_matmul(
     name: str,
 ) -> Union[TRTTensor, Sequence[TRTTensor]]:
     return impl.matmul.matrix_multiply(
-        network, target, SourceIR.ATEN, name, args[0], args[1]
+        network,
+        target,
+        SourceIR.ATEN,
+        name,
+        args[0],
+        args[1],
+        args_bounds_check(args, 2, trt.MatrixOperation.NONE),
+        args_bounds_check(args, 3, trt.MatrixOperation.NONE),
```

Comment:
You can remove these lines, since none of the converted Torch ops will have a third or fourth argument, based on their schemas.

The primary reason for adding that argument to `matrix_multiply` below is that later, for `aten.linear`, you can pass in `other_matrix_op=trt.MatrixOperation.TRANSPOSE` for instance, and then reduce the number of layers (no need for a permute)

Reply:
It makes sense! I've updated this PR and `aten.linear` in #2253.